File: February 23/13 Feb/Arithmetic Number gfg.cpp
    int inSequence(int A, int B, int C){
        // code here
        return ((B<A&&C>0 || B>A&&C<0 )?0:( (C!=0&&(B-A)%C == 0 || (B-A)==0) ? 1:0));
    }

File: February 23/13 Feb/Arithmetic Number gfg.py
    def inSequence(self, A, B, C):
        # code here
        return 0 if (B<A and C>0 or B>A and C<0) else  1 if(C!=0 and (B-A)%C == 0 or (B-A)==0) else 0;
